src/sentry/monitors/endpoints/organization_monitors.py

```diff
@@ -69,6 +69,20 @@ def get(self, request: Request, organization: Organization) -> Response:
             )
         )
         query = request.GET.get("query")
+
+        if "environment" in filter_params:
+            monitor_environments = MonitorEnvironment.objects.filter(
+                environment=filter_params["environment_objects"][0]
```

Comment:
Why do we just fetch the first one here?

Reply:
We're not going to support multiple env view. Either no env is selected or one is. Otherwise the UI gets goofy. Can throw an error if we have multiple